docs/manual/manual_dronecan.rst

```diff
@@ -44,11 +44,274 @@ Servo modules do not support DroneCAN, as shown in the table below.
+and the other reboots the device into the ST bootloader, which allows it to be programmed with new firmware. The reboot-into-bootloader can be useful for firmware updates. 
+The numbers for each are:
+
+* Normal Reboot Magic Number = 0xACCE551B1E
+* Reboot to Bootloader Magic Number = 0xBEEFCAFE
+
+The response to this request contains one boolean field. This field will be true if the number received was a valid magic number and the motor is restarting, and false if the 
+number received was not one of the valid numbers.
+
+Refer to the `uavcan.protocol.RestartNode section of Standard Data Types <https://dronecan.github.io/Specification/7._List_of_standard_data_types/#restartnode>`_ in the specification 
+for more details.
+
+uavcan.protocol.file.BeginFirmwareUpdate (Data Type ID = 40)
+#############################################################
+On modules that support DroneCAN firmware updates, this command causes the module to enter into upgrade mode, and begin attempting to firmware update over DroneCAN. 
+The request includes a source node ID and a file path on that source node. The module will attempt to get information on the file from the file server, and then begin reading the data from the upgrade file.
+
+On modules that do not support DroneCAN firmware updates, the module will respond with an INVALID_MODE error code to this request.
+
+Refer to the `uavcan.protocol.file.BeginFirmwareUpdate section of Standard Data Types <https://dronecan.github.io/Specification/7._List_of_standard_data_types/#beginfirmwareupdate>`_ in the 
+specification for more details.
+
+.. _dronecan_configuration_parameters:
+
+Configuration Parameters
+**************************
+
+Node ID
+########
+
+.. table::
+
+	+----------------+----------+
+	| **Name**       | **Type** |
+	+----------------+----------+
+	| uavcan_node_id | Integer  |
+	+----------------+----------+
+
+This parameter defines the Node ID of the module on the UAVCAN network. This ID is how the node identifies itself when sending and receiving messages. No two nodes should have 
+the same Node ID. ID 0 is typically reserved for the flight controller. A reboot is typically required after changing this parameter for the device to use the new Node ID on the network.
+
+This parameter can also be changed through the IQ Control Center if you wish to change this without using DroneCAN.
+
+.. _dronecan_bitrate_parameter:
+
+Bitrate
+########
+.. table::
+
+	+----------------+----------+
+	| **Name**       | **Type** |
+	+----------------+----------+
+	| bit_rate       | Integer  |
+	+----------------+----------+
+
+This parameter determines the DroneCAN bitrate that the module will use in bit/s. This parameter takes effect immediately when changed, so if this is changed it 
+will most likely be necessary to reconnect to the bus as at the new bitrate to continue communicating with the module. Note that most Vertiq modules do not currently 

```

Comment:
which ones do? Same as before, why don't we just list which ones do support it, and as more do, then we can add it to the list

Reply:
This sentence isn't even true anymore. Everything that supports DroneCAN supports bitrate. So I'm just taking this out.